src/main/java/game/domain/RacingGame.java

```diff
@@ -0,0 +1,53 @@
+package game.domain;
+
+import game.io.output.RacingGameOutput;
+
+import java.util.Random;
+
+public class RacingGame {
+
+    private static final Random random = new Random();
```

Comment:
정적 상수라면 대문자로 작성해주셔야 합니다 😄 
```suggestion
    private static final Random RANDOM = new Random();
```
> https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names